src/Configurator.php

```diff
@@ -8,6 +8,12 @@
 
 final class Configurator
 {
+    const CONFIG_PREFIX                     = 'config_prefix';
+    const CONFIG_SEPARATOR                  = 'config_separator';
+    const CONFIG_SERVICES_KEY               = 'services_key';
+    const CONFIG_INFLECTORS_KEY             = 'inflectors_key';
+    const CONFIG_DEFAULT_SINGLETON_SERVICES = 'default_singleton_services';
```

Comment:
@localheinz Doh! I obviously wasn't paying enough attention.

I don't regret it at all, completely the opposite - you've been extremely helpful and a pleasure to work with!